resources/puppetlabs/lein-ezbake/template/global/ext/redhat/postrm.erb

```diff
@@ -0,0 +1,18 @@
+%if %{_systemd_el}
+%systemd_postun_with_restart <%= EZBake::Config[:project] %>.service
+%endif
+%if %{_systemd_sles}
+%service_del_postun <%= EZBake::Config[:project] %>.service
+%endif
+%if %{_with_sysvinit}
+# Remove the rundir if this is an uninstall (as opposed to an upgrade)...
+if [ "$1" = "0" ]; then
+    rm -rf %{_app_rundir} || :
```

Comment:
rm -f never complains and always returns 0 so the '|| :' can be dropped without losing any function.